packages/dart_frog_test/lib/src/dart_frog_test.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:dart_frog/dart_frog.dart';
+import 'package:mocktail/mocktail.dart';
+
+class _MockRequestContext extends Mock implements RequestContext {}
+
+/// {@template dart_frog_test_context}
+/// A context used to test Dart Frog handlers.
+/// {@endtemplate}
+class DartFrogTestContext {
```

Comment:
I would make it immutable and possibly equatable

Reply:
I would love to do so, but I couldn't figure it a way of receiving the mocked dependencies in the constructor. Let me know if you have any ideas on how to do so